src/graphql/node.rs

```diff
@@ -490,8 +484,9 @@ pub enum ModuleName {
     Review,
 }
 
-pub fn is_review(hostname: &str) -> bool {
+pub fn matches_manager_hostname(hostname: &str) -> bool {
+    // Current machine's hostname is the Manager server's hostname, because this code always runs
+    // on the Manager server.
     let review_hostname = roxy::hostname();
-
     !review_hostname.is_empty() && review_hostname == hostname
```

Comment:
Since you’ve already changed the name of the method, why don't you change this `review` to `manager` as well?

Reply:
Done